state/status.go

```diff
@@ -400,21 +410,44 @@ type recordedHistoricalStatusDoc struct {
 	StatusData map[string]interface{} `bson:"statusdata"`
 }
 
+// logStatusUpdate sends the status update to the status logger.
+//
+// The "idle" status for the machine-lxd-profile is omitted from the status log,
+// since only the applied or error statuses are useful in that case.
+//
+// TODO (cderici): Once the statusesHistoryC collection goes away we'll lose
+// access to the doc parameter, so we'll replace it with a couple more
+// parameters for the status info and the status value.
+func logStatusUpdate(statusKind string, statusId string, doc statusDoc) {
+	if statusKind != "machine-lxd-profile" || doc.Status != status.Idle {
+		status_logger.InfoWithLabelsf(doc.StatusInfo, map[string]string{
```

Comment:
@cderici  Can you do a follow up and use idiomatic go (camelcase variables not snake case).